#pure_pursuit_controller:
lookahead: 1.0
speed_propotional_gain: 5.0
max_vel_x: 0.25
max_vel_theta: 2.0
min_vel_theta: -2.0
min_in_place_vel_theta: 0.5
acc_lim_theta: 3.2
acc_lim_x: .5
acc_lim_y: .5
yaw_goal_tolerance: .5
xy_goal_tolerance: .1
robot_base_frame: base_link
controller_frequency: 50
transform_tolerance: 1.0
